package.json

```diff
@@ -38,7 +38,11 @@
     "name": "Microsoft Corporation"
   },
   "license": "Apache-2.0",
+  "bin": {
+    "playwright": "./lib/cli/cli.js"
```

Comment:
Do we want to merge playwright-cli npm into playwright? I thought we wanted to keep them separate.

Reply:
I think that's the point of merging!